scripts/install

```diff
@@ -103,8 +103,9 @@ sudo yunohost app ssowatconf
 # Wordpress installation
 curl -kL -H "Host: $domain" --data "&weblog_title=YunoBlog&user_name=$admin_wordpress&admin_password=$db_pwd&admin_password2=$db_pwd&admin_email=$admin_wordpress@$domain&language=$language&Submit=Install+WordPress" https://localhost$path/wp-admin/install.php?step=2 > /dev/null 2>&1
 
-for i in `seq 1 30`
-do	# La boucle attend la fin de l'installation de wordpress Ou 30 secondes.
+echo -n "Please wait during Wordpress installation" >&2
+for i in `seq 1 300`
```

Comment:
I think it could be a good idea to check regularly the installation, for example by asking database or by checking existing file.